AppCenterCrashes/AppCenterCrashesTests/MSACCrashesUtilTests.m

```diff
@@ -110,14 +110,14 @@ - (void)testCreateWrapperExceptionDir {
 // Before SDK 12.2 (bundled with Xcode 10.*) when running in a unit test bundle the bundle identifier is null.
 // 12.2 and after the above bundle identifier is com.apple.dt.xctest.tool.
 - (NSString *)getPathWithBundleIdentifier:(NSString *)path {
-    NSString* bundleId;
-#if ((defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 120200) || \
-    (defined(__MAC_OS_X_VERSION_MAX_ALLOWED) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 101404))
-    bundleId = @"com.apple.dt.xctest.tool";
+  NSString *bundleId;
+#if ((defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 120200) ||                                            \
+     (defined(__MAC_OS_X_VERSION_MAX_ALLOWED) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 101404))
```

Comment:
I know, it's not the change of this PR, but..
Could you please use constants here?